### ISSUE TITLE
[Bug]: Runtime exception due to pydantic version(v2.0) mismatch

### ISSUE THREAD

**Issue by ZhUyU1997** [towhee-io/towhee#2610]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

```python
from towhee import pipe

add_one = (
    pipe.input('x')
        .map('x', 'y', lambda x: x + 1)
        .output('y')
)

res = add_one(0).get()
```
Get exception:
```
(VENV) D:\Milvus\prj>python ./main.py
Traceback (most recent call last):
  File "D:\Milvus\prj\main.py", line 1, in <module>
    from towhee import pipe
  File "D:\Milvus\prj\VENV\lib\site-packages\towhee\__init__.py", line 20, in <module>
    from towhee.runtime import register, pipe, ops, accelerate, AutoConfig, AutoPipes
  File "D:\Milvus\prj\VENV\lib\site-packages\towhee\runtime\__init__.py", line 17, in <module>
    from .pipeline import Pipeline as pipe
  File "D:\Milvus\prj\VENV\lib\site-packages\towhee\runtime\pipeline.py", line 18, in <module>
    from towhee.runtime.check_utils import TupleForm
  File "D:\Milvus\prj\VENV\lib\site-packages\towhee\runtime\check_utils.py", line 36, in <module>
    class TupleForm(BaseModel):
  File "D:\Milvus\prj\VENV\lib\site-packages\towhee\runtime\check_utils.py", line 38, in TupleForm
    schema_data: Optional[Tuple[constr(regex='^[a-z][a-z0-9_]*$'), ...]]
TypeError: constr() got an unexpected keyword argument 'regex'
```

### Expected Behavior

No exception.

### Steps To Reproduce

```markdown
1. Follow the guide of https://docs.towhee.io/Getting%20Started/install/#install-towhee-with-pip
2. Write examples then run

from towhee import pipe

add_one = (
    pipe.input('x')
        .map('x', 'y', lambda x: x + 1)
        .output('y')
)

res = add_one(0).get()
```

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93): v1.1.0
- OS(Ubuntu or CentOS): Windows
- CPU/Memory: i7/32G
- GPU: 4070Ti
- Others:
```

**Comment by junjiejiangjjj** [comment]
We have released Towhee 1.1.1, it uses the correct version of `pydantic`

**Comment by xxxmoonquake-dxq** [comment]
请问您是怎么解决这个问题的
